### PR TITLE
fix(rnd): Disable unused prisma connection on pyro API Server process

### PR DESCRIPTION
**Pull request by majdyz**
### Background

Pyro for API Server is not using Prisma, but still holding a Prisma connection.
The fast-API thread is also holding a Prisma connection, making Prisma connected in two different loop within a single process.

### Changes 🏗️

Disable a Prisma connection on Pyro thread for Server API process.

### PR Quality Scorecard ✨

- [x] Have you used the PR description template? &ensp; `+2 pts`
- [ ] Is your pull request atomic, focusing on a single change? &ensp; `+5 pts`
- [ ] Have you linked the GitHub issue(s) that this PR addresses? &ensp; `+5 pts`
- [ ] Have you documented your changes clearly and comprehensively? &ensp; `+5 pts`
- [ ] Have you changed or added a feature? &ensp; `-4 pts`
  - [ ] Have you added/updated corresponding documentation? &ensp; `+4 pts`
  - [ ] Have you added/updated corresponding integration tests? &ensp; `+5 pts`
- [ ] Have you changed the behavior of AutoGPT? &ensp; `-5 pts`
  - [ ] Have you also run `agbenchmark` to verify that these changes do not regress performance? &ensp; `+10 pts`

### CODE REVIEW

**Comment by majdyz** [comment]
@Bentlybro not related or going to fix the frontend issue, but I see some request are sometimes failing due to this.

**Review by Bentlybro — APPROVED** [review]
tested and works :) - tho this broke the pipeline?

File: rnd/autogpt_server/autogpt_server/executor/manager.py [hunk 1]
```diff
@@ -173,40 +175,41 @@ def register_next_executions(node_link: Link) -> list[NodeExecution]:
                 )
             )
 
-        # Complete missing static input pins data using the last execution input.
```
Comment (majdyz) [inline rnd/autogpt_server/autogpt_server/executor/manager.py:176]: No change, only tabbing and add more comments

**Comment by majdyz** [comment]
@Bentlybro fixed the failing pipeline, would you mind re-approving ? :)

**Comment by Bentlybro** [comment]
@majdyz sure, tho curious on the revert of "Fix(Builder): Copy and Paste when editing textbox causes issue"?

**Review by Bentlybro — APPROVED** [review]

**Pull request merged**